WBooks/Book Table/BookTableAbstractViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  BookTableAbstractViewModel.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 13/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveSwift
+
+class BookTableAbstractViewModel {
```

Comment:
Maybe this could be a protocol, not a class

Reply:
I had decided on a class so I could better reuse the instance members `books` and `state` in the subclasses. On the other hand this was intended to be an abstract class with an abstract function `loadBooks()`, but swift does not support that. So a protocol does sound more Swift like. I'll change that!